src/fparser/two/Fortran2008.py

```diff
@@ -96,6 +101,241 @@ class Program_Unit(Program_Unit_2003):  # R202
     subclass_names.append("Submodule")
 
 
+class Component_Attr_Spec(STRINGBase):  # R437
+    """
+:F08R:`437`
```

Comment:
Could you please use the newer Formatting for docstrings (take a look at the other routines in the Fortran2008.py classes for examples).